source/local/local.go

```diff
@@ -113,6 +113,7 @@ func (s *localSource) Discover() error {
 	if s.config.HooksEnabled {
 
 		klog.InfoS("starting hooks...")
+		klog.InfoS("hooks are DISABLED by default and will be removed in a future release.")
```

Comment:
This message looks a bit strange, user might not be interested what's the default, but might be interested that they're deprecated

```suggestion
		klog.InfoS("NOTE: hooks are deprecated and will be completely removed in a future release.")
```